SPECS/hyperv-daemons/hyperv-daemons.spec

```diff
@@ -8,14 +8,15 @@
 %global udev_prefix 70
 Summary:        Hyper-V daemons suite
 Name:           hyperv-daemons
-Version:        5.4.91
+Version:        5.10.13.1
 Release:        1%{?dist}
 License:        GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System/Kernel
-URL:            https://github.com/microsoft/WSL2-Linux-Kernel
-Source0:        https://github.com/microsoft/WSL2-Linux-Kernel/archive/linux-msft-%{version}.tar.gz
+URL:            https://github.com/microsoft/mariner-linux-kernel
+#Source0:        https://github.com/microsoft/mariner-linux-kernel/archive/rolling-lts/mariner/%{version}.tar.gz
+Source0:        kernel-%{version}.tar.gz
```

Comment:
Is this needed since the source URL doesn't work right now?

Reply:
Unfortunately yes. When we download the source tarball from https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner/5.10.13.1.tar.gz , this will yield `5.10.13.1.tar.gz`. Because this tarball is only a version number, there are possible naming collisions on the source server.

When we had situations like this before, we resolved it by adding the package name to the front of the tarball and annotated the spec file shown here so the resulting Source0 used by rpmbuild is `kernel-5.10.13.1.tar.gz`, but one can directly obtain the tarball from the commented Source0 URL. The sha256 hash of the tarball is stored in `kernel.signatures.json` so any new download from the URL can be independently verified.